recipes/sorcha-addons/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "sorcha-addons" %}
+{% set version = "0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dirac-institute/sorcha-addons/archive/v{{ version }}.tar.gz
+  sha256: bd9c980ddc679a7cf9be00eb3aeeca2582ae8483902532f3df49b1563170f424
+
+build:
+#  noarch: python
```

Comment:
Why not noarch?

Reply:
staged-recipes wouldn't build, unfortunately -- it keeps trying to build the windows version, which fails because it doesn't have the necessary (platform-specific) dependencies (specifically, `sorcha`).

And the linter stops the CI from passing if `noarch` is present with a platform selector.

Reply:
PS: Happy to (would like to!) change this to noarch if there's a known workaround!

Reply:
Failing to build due to a missing dependency is fine. We prefer noarch to save CI resources and, when this dependency is available on Windows, this package would be ready without the need for any modifications.

Reply:
Oh, didn't realize you could ask for review while the CI isn't passing (for any reason)! I'll go ahead and uncomment that line & resubmit. There in ~5 minutes.

Reply:
Noarch with failing dependencies on a certain arch is ok b/c the package won't be installable in that arch until the dependency is available. So, in a way, it is not "broken" but "pending a dependency. Anyway, it is not something to be fixed in this package but in the dependency.